rubin_sim/maf/maps/create_gaia_density_map.py

```diff
@@ -106,18 +112,21 @@
         results = gaia_db.get_arbitrary_chunk_iterator(query, dtype=dtype, chunk_size=10000)
         result = list(results)[0]
 
-        distances = angular_separation(result["ra"], result["dec"], ra[i], dec[i])  # Degrees
+        distances = angular_separation(result["ra"], result["dec"], ra[i], dec[i])
         result = result[np.where(distances < radius)]
 
         import pdb
```

Comment:
Urg.  I don't actually know what the problem was, why there was a pdb set in here. I suspect it was just that the script was sometimes timing out (I never ran this). 
It's also worth noting that this will not run anymore -- the machine and framework it was meant to run against ceased to exist. Hopefully we just never have to recreate the Gaia map. In some aspects, it's here because it helps us know how the maps we're using were created .. we probably ought to make a legacy sims repo to host stuff like this instead (along with the FOV-map code, etc.).

Reply:
I pulled out the pdb lines.